Makefile

```diff
@@ -13,7 +13,8 @@ user-manual-html:
 	cd user_manual && make html
 	@echo "User manual build finished; HTML is updated"
 
-developer-manual-html: icons-docs
+# Should also build icons-docs, but due to https://github.com/nextcloud/documentation/issues/7162 it is disabled
+developer-manual-html: openapi-spec #icons-docs
```

Comment:
```suggestion
developer-manual-html: openapi-spec icons-docs
```

*cough*

Reply:
I know, but the icons-docs are broken. Easier to disable this and keep the php8 from openapi-extractor.

Reply:
Not sure what the builder that does it for the releases has installed